pkg/client/compose/config.go

```diff
@@ -71,5 +78,11 @@ func configSpecsFromCompose(
 		configMounts = append(configMounts, mount)
```

Comment:
Just found out that `ContainerSpec.Equal` depends on the order of config mounts. This is not a big deal as usually people update and deploy the same Compose file so the order of `service.configs` normally stays the same hence the spec comparison works as expected.

However, while we're on this, it would be good to make the comparison order agnostic, like with volume mounts: https://github.com/psviderski/uncloud/blob/79dc05cb665d6541ea2ff1ca16af4119ae6b47d4/pkg/api/service.go#L289-L300